docs/install/install-on-kubernetes.md

```diff
@@ -23,35 +23,35 @@ helm show values infrahq/infra
 server:
   # Add an Identity Provider
   # Only Okta is supported currently
-  additionalProviders:
-    - name: Okta
-      url: example.okta.com
-      clientID: example_jsldf08j23d081j2d12sd
-      clientSecret:  example_plain_secret #see note above
+  # additionalProviders:
+  #   - name: Okta
+  #     url: example.okta.com
+  #     clientID: example_jsldf08j23d081j2d12sd
+  #     clientSecret:  example_plain_secret # see note above
+
+  # Add an admin user
```

Comment:
```suggestion
  # Add a user
```
This wording implies that any identity in the `additionalIdentities` is an admin to me.